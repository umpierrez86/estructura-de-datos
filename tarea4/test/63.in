# implementar mezclaCadenas
mezclaCadenas 0 0
mezclaCadenas 0 1 (50,0)
mezclaCadenas 1 (50,0) 0
mezclaCadenas 1 (40,0) 1 (50,0)
mezclaCadenas 1 (40,0) 1 (30,0)
mezclaCadenas 1 (40,3.0) 1 (40,0)
mezclaCadenas 1 (90,0) 4 (10,0) (20,0) (30,0) (40,0)
mezclaCadenas 4 (10,0) (20,0) (30,0) (40,0) 1 (90,0)
mezclaCadenas 3 (70,0) (80,0) (90,0) 4 (10,0) (20,0) (30,0) (40,0)
mezclaCadenas 4 (10,0) (20,0) (30,0) (40,0) 3 (70,0) (80,0) (90,0)
mezclaCadenas 3 (70,0) (80,0) (90,0) 4 (65,0) (75,0) (85,0) (95,0)
mezclaCadenas 4 (65,0) (75,0) (85,0) (95,0) 3 (70,0) (80,0) (90,0)
mezclaCadenas 4 (20,1) (30,0) (40,3) (50,3) 3 (20,0) (35,0) (50,0)
mezclaCadenas 4 (20,1) (30,0) (40,3) (50,3) 3 (25,0) (40,0) (45,0)
mezclaCadenas 4 (20,1) (30,0) (40,3) (50,3) 3 (15,0) (30,0) (55,0)
Fin