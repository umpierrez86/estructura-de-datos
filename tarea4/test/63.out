1>#  implementar mezclaCadenas.
2>
3>(50,0.00)
4>(50,0.00)
5>(40,0.00)(50,0.00)
6>(30,0.00)(40,0.00)
7>(40,3.00)
8>(10,0.00)(20,0.00)(30,0.00)(40,0.00)(90,0.00)
9>(10,0.00)(20,0.00)(30,0.00)(40,0.00)(90,0.00)
10>(10,0.00)(20,0.00)(30,0.00)(40,0.00)(70,0.00)(80,0.00)(90,0.00)
11>(10,0.00)(20,0.00)(30,0.00)(40,0.00)(70,0.00)(80,0.00)(90,0.00)
12>(65,0.00)(70,0.00)(75,0.00)(80,0.00)(85,0.00)(90,0.00)(95,0.00)
13>(65,0.00)(70,0.00)(75,0.00)(80,0.00)(85,0.00)(90,0.00)(95,0.00)
14>(20,1.00)(30,0.00)(35,0.00)(40,3.00)(50,3.00)
15>(20,1.00)(25,0.00)(30,0.00)(40,3.00)(45,0.00)(50,3.00)
16>(15,0.00)(20,1.00)(30,0.00)(40,3.00)(50,3.00)(55,0.00)
17>Fin.
